test/support/factory.ex

```diff
@@ -150,15 +150,31 @@ defmodule Wanda.Factory do
     execution_id = Map.get(attrs, :execution_id, Faker.UUID.v4())
     group_id = Map.get(attrs, :group_id, Faker.UUID.v4())
 
+    payload =
```

Comment:
We could use an empty payload by default and add a `with_payload` helper function to the factory: 
https://github.com/thoughtbot/ex_machina#flexible-factories-with-pipes